gallery/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounterPage.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTextLink, NxTable, NxP, NxCode } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxSmallVulnerabilityCounterExample from './NxSmallVulnerabilityCounterExample';
+import NxSmallVulnerabilityCounterCustomExample from './NxSmallVulnerabilityCounterCustomExample';
+import NxSmallVulnerabilityCounterInfiniteMaxDigitsExample from './NxSmallVulnerabilityCounterInfiniteMaxDigitsExample';
+
+const NxSmallVulnerabilityCounterCode = require('./NxSmallVulnerabilityCounterExample?raw');
+const NxSmallVulnerabilityCounterCustomCode = require('./NxSmallVulnerabilityCounterCustomExample?raw');
+const NxSmallVulnerabilityCounterInfiniteMaxDigitsCode = require(
+    './NxSmallVulnerabilityCounterInfiniteMaxDigitsExample?raw');
+
+const NxSmallVulnerabilityCounterPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
```

Comment:
There's no mention of CVSS in this description.  I think there should be, as this component is intended to count vulnerabilities with CVSS scores that fall into each of the different CVSS score ratings.

Reply:
Mentioned CVSS score ratings on: [5db5cc7](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/5db5cc7d035df0b95746ccbbbcb61754a3a59e50)